src/Runtimes/Runtimes.php

```diff
@@ -16,8 +16,9 @@ class Runtimes
     public function __construct()
     {
         $node = new Runtime('node', 'Node.js');
-        $node->addVersion('14.5', 'node:14.5-alpine', 'appwrite/runtime-for-node:14.5', [System::X86, System::PPC, System::ARM]);
-        $node->addVersion('15.5', 'node:15.5-alpine', 'appwrite/runtime-for-node:15.5', [System::X86, System::PPC, System::ARM]);
+        $node->addVersion('14.5', 'node:14.5-alpine', 'appwrite/env-node-14.5:1.0.0', [System::X86, System::PPC, System::ARM]);
+        $node->addVersion('15.5', 'node:15.5-alpine', 'appwrite/env-node-15.5:1.0.0', [System::X86, System::PPC, System::ARM]);
+        $node->addVersion('16', 'node:16-alpine', 'appwrite/env-node-16:1.0.0', [System::X86, System::PPC, System::ARM]);
```

Comment:
I think it would make more sense to be more persistent and use node `16.0` as the version number instead of just `16`

Reply:
@TorstenDittmann do you agree?

Reply:
I would wait until we have proper docker hub versions and use 16.0.